ED/src/dynamics/mortality.f90

```diff
@@ -273,16 +279,26 @@ real function survivorship(new_lu,dist_path,mindbh_harvest,cpatch,ico)
 
        case (6)
          !---------------------------------------------------------------------------------!
-         !     Logging.  At this point a single pathway exists: cohorts above threshold    !
+         !     Logging.                                                                    !
+         ! NEW: Survivorship = inverse fraction of amount removed above and below the hite !
+         !      threshold.  This allows for understory thinning as a management strategy   !
+         !                                                                                 !
+         !---------------------------------------------------------------------------------!
+         ! OLD: At this point a single pathway exists: cohorts above threshold             !
          ! are completely removed, and small cohorts have the same survivorship as small   !
          ! cohorts at a treefall site.  Both could be re-visited in the future, and        !
          ! different pathways for conventional and reduced-impact logging could be         !
          ! applied.                                                                        !
          !---------------------------------------------------------------------------------!
          if (cpatch%dbh(ico) >= mindbh_harvest(ipft)) then
-            survivorship = 0.0
+            !if (ianth_disturb == 2) then
+            	survivorship = 1 - harvprob_g(ipft)
+            ! else 
+            !	survivorship = 0.0
+            ! end if 
```

Comment:
@crollinson I am going through the code merging but this part is a bit confusing to me., and I think this approach here may be saving too many trees.  In my original implementation, harvprob is used to define the disturbance rate.  In general, survivorship should reflect the survivorship in the immediate vicinity of disturbed area (in this case, where the tree was logged), which is why survivorship was set to zero.  For the time being I kept your approach for the area-based option (ianth_disturb=2), but I am not sure how to reconcile this part with the original survivorship definition, especially with understory thinning.

Reply:
The only idea I had so far is to create a new dist_type for thinned canopy.  The upside is that it would be consistent with the ED idea that disturbances change the microenvironment (in the case of thinning, more in the water part than in the light).  The one potential downside is that we would have more dist_types (tree felling, skid trails and roads, and artificially thinned canopy).

I don't see a problem to add a new dist_type if it is justified, but before I go ahead, I'd like to throw the idea and see what people think, or if anyone has other ideas.